imports/ui/explorer-components/filter-row.jsx

```diff
@@ -1,38 +1,118 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import {
+  selectInputIconStyle,
+  noUnderline,
+  sortBoxInputStyle,
+  selectedItemStyle,
+  sortBoxBlueInputStyle
+} from '../components/form-controls.mui-styles'
+import MenuItem from 'material-ui/MenuItem'
+import SelectField from 'material-ui/SelectField'
+import { labels } from '../explorer-components/sort-items'
 
 export class FilterRow extends Component {
+  constructor (props) {
+    super(props)
+    this.handleStatusFilterClicked = this.handleStatusFilterClicked.bind(this)
+    this.handleRoleFilterClicked = this.handleRoleFilterClicked.bind(this)
+    this.handleSortClicked = this.handleSortClicked.bind(this)
+  }
+
+  handleStatusFilterClicked (event, index, values) {
```

Comment:
change to `handleStatusFilterClicked = (event, index, values) => {` so you won't need to use the pattern trick I mentioned above. Do the same for the other handler methods